tests/lob.phpt

```diff
@@ -0,0 +1,126 @@
+--TEST--
+pdo_snowflake - max lob size
+--INI--
+pdo_snowflake.cacert=libsnowflakeclient/cacert.pem
+--FILE--
+<?php
+    set_time_limit(180);
```

Comment:
let's set the timeout to 720 - I tested that 180 is not enough for this test :(

Reply:
done